src/sos/__main__.py

```diff
@@ -912,7 +912,6 @@ def cmd_worker(args, workflow_args):
         from .workers import SoS_Worker
 
         procs = [SoS_Worker(env.config) for i in range(args.workers)]
-        [p.start() for p in procs]
```

Comment:
How can you remove the statement? As I explained to you before, this expression has the side effect of starting all the `p`. To remove the warning, you can change it to 

```
for p in proces:
    p.start()
```